test/sai_test/config/route_configer.py

```diff
@@ -314,6 +409,47 @@ def create_neighbor_by_rif(self, nexthop_device: Device, rif, no_host=True):
             self.test_obj.dut.neighborv6_list.append(nbr_entry_v6)
         return nbr_entry_v4, nbr_entry_v6
 
+    def create_nexthop_group_by_nexthops(self, nexthopv4_list: List[Nexthop], nexthopv6_list: List[Nexthop], lag_list: List[Lag], dest_device: Device):
```

Comment:
This method will create nexthop base on lag, please specified it from method name

Reply:
No, this method creates nexthop group based on nexthops, (i.e., `nexthopv4_list` and `nexthopv6_list`). Passing `lag_list` is to assign the values to `nexthop_groupv4` and `nexthop_groupv6`, which are new added in the `Lag` class.

Reply:
assign it outside this method? then this create nexthop group just will be more specific